src/modules/autocomplete.js

```diff
@@ -107,7 +107,7 @@ class Autocomplete {
    * @param {number} [userParameters.clientId] - Client ID, utilized to personalize results
    * @param {string} [userParameters.userId] - User ID, utilized to personalize results
    * @param {string} [userParameters.segments] - User segments
-   * @param {string} [userParameters.testCells] - User test cells
+   * @param {object} [userParameters.testCells] - User test cells ex: { 'cellName' : 'cellValue' }
```

Comment:
Thanks for handling this! Can we change these to  `{ 'testName' : 'cellName' }` and move it to the example block we have on line 117.

Reply:
Sounds good. Should I include it into the example call like with better formatting of course? Or just a separate example section.

   * constructorio.autocomplete.getAutocompleteResults('t-shirt', {
   *     resultsPerSection: {
   *         Products: 5,
   *         'Search Suggestions': 10,
   *     },
   *     filters: {
   *         size: 'medium'
   *     },
   * }, { testCells: { 'testName' : 'cellName' }});